evmcc/evmcc_test.go

```diff
@@ -750,6 +753,7 @@ Vc4foA7mruwjI8sEng==
 
 						Expect(stub.SetEventCallCount()).To(Equal(initialEventCallCount + 1))
 						setEventName, setEventPayload := stub.SetEventArgsForCall(initialEventCallCount)
+						Expect(utf8.ValidString(setEventName)).To(BeTrue())
```

Comment:
I'm not worried about this anymore. Would be better to program the stub to do this check.

Reply:
We could add it to the stub on the SetEvent, so that it throws an error/panics like the stub would in the peer